src/librustc_macros/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "rustc_macros"
+version = "0.1.0"
+authors = ["The Rust Project Developers"]
```

Comment:
Is it possible to make this a rust 2018 crate? Otherwise, somebody will come along and do that later for #58099 anyway, leading to more churn...

Reply:
we could/should enforce this via tidy once we have all of them ported